src/components/SidebarNew/Bar.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { barStyle } from './Bar.style'
+import HomeIcon from '@mui/icons-material/Home';
+import ChatBubbleIcon from '@mui/icons-material/ChatBubble';
+import GroupsIcon from '@mui/icons-material/Groups';
+import LogoutIcon from '@mui/icons-material/Logout';
+import { Typography, Box } from '@mui/material';
+import { Link } from 'react-router-dom';
+import AccountBoxIcon from '@mui/icons-material/AccountBox';
+
+const Bar = () => {
+  const {classes}= barStyle();
+
+  return (
+  <Box className={classes.mainContainer}>
+    <Box className={classes.containerOne}>
```

Comment:
here also the below inner component is getting repeated, use an array and a map function

Reply:
Changed to an array and rendered with a map method !